src/DataViews/SamplingReportDataView.cpp

```diff
@@ -340,6 +339,34 @@ void SamplingReportDataView::SetStackEventsCount(uint32_t stack_events_count) {
   stack_events_count_ = stack_events_count;
 }
 
+std::string SamplingReportDataView::GetToolTip(int row, int column) {
+  if (column != kColumnInclusive && column != kColumnExclusive) {
+    return "";
+  }
+  const SampledFunction& function = GetSampledFunction(row);
+  uint32_t raw_count = {};
+  float percentage = {};
+  if (column == kColumnInclusive) {
+    raw_count = function.inclusive;
+    percentage = function.inclusive_percent;
+  } else {
+    raw_count = function.exclusive;
+    percentage = function.exclusive_percent;
+  }
+
+  orbit_statistics::BinomialConfidenceInterval interval =
+      app_->GetConfidenceIntervalEstimator().Estimate(percentage / 100.0f, stack_events_count_);
+
+  return absl::StrFormat(
+      "The function %s\n"
```

Comment:
```suggestion
      "The function \"%s\"\n"
```

Reply:
Don't think that would make the notation cluttered?

Reply:
This is how we usually log function names. I don't have a strong opinion about tooltips.